opensrp-giz-malawi/src/main/java/org/smartregister/giz/task/SaveReasonForDefaultingEventTask.java

```diff
@@ -0,0 +1,131 @@
+package org.smartregister.giz.task;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.smartregister.child.ChildLibrary;
+import org.smartregister.child.util.ChildJsonFormUtils;
+import org.smartregister.child.util.Constants;
+import org.smartregister.clientandeventmodel.Event;
+import org.smartregister.clientandeventmodel.FormEntityConstants;
+import org.smartregister.domain.Client;
+import org.smartregister.domain.db.EventClient;
+import org.smartregister.giz.application.GizMalawiApplication;
+import org.smartregister.repository.EventClientRepository;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Iterator;
+
+import timber.log.Timber;
+
+public class SaveReasonForDefaultingEventTask extends AsyncTask<Void, Void, Void> {
```

Comment:
Though we stopped using `AsyncTask`. Could we replace this with AppExecutors. What do you think?

Reply:
Removed